asllib/doc/Types.tex

```diff
@@ -61,6 +61,17 @@ \subsection{Abstract Syntax}
 
 We define $\buildtydecl$ per the corresponding type in the following sections.
 
+\hypertarget{build-tyorcollection}{}
+The function
+\[
+  \buildtyorcollection(\overname{\parsenode{\Ntyorcollection}}{\vparsednode}) \;\aslto\; \overname{\ty}{\vastnode}
+  \cup \overname{\TBuildError}{\BuildErrorConfig}
+\]
+transforms a type annotation parse node $\vparsednode$ into a type AST node $\vastnode$.
+\ProseOtherwiseBuildError
+
+We define $\buildtyorcollection$ per the corresponding type in the following sections.
```

Comment:
I'm not sure I follow where the rules for this are - I can see the `collection` case, but not the `ty` case